client/packages/invoices/src/utils.ts

```diff
@@ -226,6 +226,7 @@ export const canReturnInboundLines = (inbound: InboundFragment): boolean =>
   inbound.status === InvoiceNodeStatus.Verified;
 
 export const canReturnOutboundLines = (outbound: OutboundFragment): boolean =>
+  outbound.status === InvoiceNodeStatus.Shipped ||
```

Comment:
If a manual outbound shipment, it never goes beyond shipped, but the supplier should be able to process the return of those goods!

Reply:
likely worth adding the check in then - only if manual then allow `Shipped` to be returned. As it is, a customer can return stock that is in transit 🤔

Reply:
Curious for more thoughts on this one... this is for processing the return of goods from an outbound shipment - i.e. a supplier saying yes I shipped these goods but now they are being returned. I'd guess that most of the time, inbound returns would be generated from outbound returns (created by the customer), but where you did want to create one manually, would it be worth constraining that you can't do that until the customer marks that shipment as delivered?